deploy/service.yaml

```diff
@@ -17,3 +17,4 @@ spec:
       name: prometheus
   selector:
     app: 3scale-istio-adapter
+  clusterIP: None
```

Comment:
note: this will require an update to Maistra templates

Reply:
Sure. Will submit the change or ping them on merge